docs/BR.md

```diff
@@ -301,6 +304,8 @@ The Definitions found in the CA/Browser Forum's Network and Certificate System S
 
 **Application Software Supplier**: A supplier of Internet browser software or other relying-party application software that displays or uses Certificates and incorporates Root Certificates.
 
+**Address and Routing Parameter Area Name**: A Domain Name whose Top-Level Domain is "arpa". Example: `1.1.168.192.in-addr.arpa`.
```

Comment:
The language is clear but perhaps the example makes the reader believe we are trying to block only non-routable IP addresses. If this is not a big concern by others, you may ignore this comment :)

Reply:
That's a good point. Maybe we can switch to an IP address from one of these blocks: https://datatracker.ietf.org/doc/html/rfc5737#section-3